ftl/python/builder.py

```diff
@@ -68,12 +70,13 @@ def Build(self):
 
             with ftl_util.Timing("resolving whl paths"):
                 whls = self._resolve_whls()
+                logging.info("WHLS:\n" + str(whls))
```

Comment:
You can use string formatting to make this cleaner.

Reply:
removing this statements as they were for debugging